frontend/src/components/match/FinalSpiritScores.js

```diff
@@ -0,0 +1,52 @@
+import clsx from "clsx";
+import { Icon } from "solid-heroicons";
+import { chevronRight } from "solid-heroicons/solid";
+
+import Modal from "../Modal";
+
+const FinalSpiritScoresModal = props => {
```

Comment:
same here... atleast in react this is not good practice as it will confuse during import and export

Reply:
done